extra-dev/packages/coq-ext-lib/coq-ext-lib.dev/opam

```diff
@@ -16,7 +16,7 @@ remove: [
 ]
 depends: [
   "ocaml"
-  "coq" {= "dev"}
+  "coq"
```

Comment:
Does this package really work with _all_ versions of Coq, past and present? If not, I would recommend a disjunction that includes both `dev` and some stable versions, e.g., `{(>= "8.7" & < "8.10~") | (= "dev")}`. Even an upwards-open interval of stable versions would be preferable: `{(>= "8.7") | (= "dev")}`